content/english/python-pixel/B.conditions&loops/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Python: conditions & loops"
```

Comment:
This file should also be moved to the Python basics section, like the nested-loops file.